app/setup.py

```diff
@@ -448,11 +463,14 @@ def get_locale():  # pylint: disable=unused-variable
 
         return None
 
-    @application.babel.timezoneselector
     def get_timezone():  # pylint: disable=unused-variable
         # For now regardless of locale we will show times in GMT/BST
         return "Europe/London"
 
+    application.babel.init_app(
```

Comment:
Can you provide link to the doc, that shows this update?.

Reply:
https://pyup.io/packages/pypi/flask-babel/changelog#:~:text=%2D%20Babel.locale_selector%20and%20Babel.timezone_selector%20no%20longer%20exist.%20They%20must%20be%20provided%0Aeither%20when%20the%20Babel()%20object%20is%20created%20or%20when%20init_app()%20is%20called.%20This%20is%20to%20support%0Ahaving%20a%20single%20Babel%20object%20for%20multiple%20Flask%20apps%20(107)%20as%20well%20as%20to%20simplify%20settings%0Aand%20multi%2Dthreaded%20state.